### ISSUE TITLE
Is this library still maintained?

### ISSUE THREAD

**Issue by edoantonioco** [livechat/react-native-livechat#45]
This library seems to be great, but it has many issues, and I get the impression that is not maintained anymore.

The bug of keyboard being on top of the text input is present, which is fixed by this https://github.com/FaridSafi/react-native-gifted-chat/issues/680#issuecomment-377662861

I get this error:

> Setting a timer for a long period of time, i.e. multiple minutes, is a performance and correctness issue on Android as it keeps the timer module awake, and timers can only be called when the app is in the foreground. See https://github.com/facebook/react-native/issues/12981 for more info.
(Saw setTimeout with duration 1800000ms)

And also this one

> Warning: ListView is deprecated and will be removed in a future release. See https://fb.me/nolistview for more information

Also in Chat.js line 177, there is this line
`height: Platform.OS === 'ios' ? height : height - height / 25,`
which causes issues with other navigation headers, changing it with
`height: '100%'`
can be an option.

So my request is that this library still get maintained, since its great.

**Comment by Alenius** [comment]
Agreed. I have some problems as well, an example of this is that I cannot see the chat bubbles at all when chatting...

**Comment by IsaacHardy** [comment]
@Alenius I'm currently attempting to resolve this issue. I have a feeling it has to do with an outdated `livechat-vistor-sdk`.

**Comment by IsaacHardy** [comment]
To be honest, this package is so old most all of its dependencies need updating.

**Comment by mm7mod** [comment]
having the same issue/feeling ....most dependencies are very old and now with react-native .60 this library  crashes from the beginning as there are features such as list view no longer supported .

**Comment by Alenius** [comment]
@IsaacHardy I have tried to connect to utilize the sdk via the .min.js-file but I can't even use init for some reason, have you succeeded? I really want this to work, even if I have to implement everything from scratch by myself except the sdk connection.

**Comment by creative-git** [comment]
i replaced the SDK with version 0.35.2. and it compiles without any issue. however, the messages are not sent. ( no matter what sdk version sdk i use) . any idea ?

**Comment by Volekss** [comment]
Same here. Any forks that solve the issue with sending messages?

**Comment by IsaacHardy** [comment]
@Alenius I'm in the same boat still. I've attempted to update the necessary packages and I've even replaced the `.min.js` file with the latest with no avail. 

I've decided to suggest a different solution for my client until I can spend more time "resolving" LiveChat's own product package.

**Comment by mm7mod** [comment]
@IsaacHardy  "I've decided to suggest a different solution for my client until I can spend more time "resolving" LiveChat's own product package. "  did the same here xD xD 

the SDK doens't seem to send messages with me

**Comment by creative-git** [comment]
same here... i abandoned this library .. its useless
